app/src/scenes/Home/index.tsx

```diff
@@ -27,6 +28,9 @@ class Home extends React.Component {
               <img width="80%" src={heroImg} alt="Hero image" />
             </Col>
           </Row>
+          <Col md={22}>
```

Comment:
I think we don't need this col. Isn't it?

Reply:
No